src/config.ts

```diff
@@ -164,3 +180,65 @@ export function getGitTagPrefix(): string {
   const githubTarget = targets.find(target => target.name === 'github') || {};
   return githubTarget.tagPrefix || '';
 }
+
+/**
+ * Loads environment variables from ".craft.env" files in certain locations
+ *
+ * The following two places are checked:
+ * - The user's home directory
+ * - The current working directory
+ *
+ * @param overwriteExisting If set to true, overwrite the existing environment variables
+ */
+export function readEnvironmentConfig(
+  overwriteExisting: boolean = false
+): void {
+  let newEnv = {} as any;
+
+  // Read from home dir
+  const homedirEnvFile = join(homedir(), ENV_FILE_NAME);
+  if (existsSync(homedirEnvFile)) {
+    logger.info(
+      `Found environment file in the home directory: ${homedirEnvFile}`
+    );
+    const homedirEnv = {};
+    nvar({ path: homedirEnvFile, target: homedirEnv });
+    newEnv = { ...newEnv, ...homedirEnv };
+    logger.debug(
+      `Read the following variables from ${homedirEnvFile}: ${Object.keys(
+        homedirEnv
+      )}`
+    );
+  } else {
+    logger.debug(
+      `No environment file found in the home directory: ${homedirEnvFile}`
+    );
+  }
+
+  // Read from current dir
+  const currentDirEnvFile = join(process.cwd(), ENV_FILE_NAME);
+  if (existsSync(currentDirEnvFile)) {
```

Comment:
This check is prone to race conditions. You could probably just invoke `nvar` and then catch down the expected "file doesn't exist" exception. Also, this would allow to explicitly handle the case when the file is not readable.

Reply:
TBH, I don't really want to worry too much about what can go wrong here (file not readable, it's a directory, etc.)